.travis.yml

```diff
@@ -54,5 +66,9 @@ script:
         export PKG_CONFIG_PATH=$(brew --prefix openssl)"/lib/pkgconfig";
         cmake . -DCMAKE_BUILD_TYPE=$BUILD_TYPE $BUILD_OPTS -DENABLE_UNITTESTS="ON";
       fi
-    - make
+    - if [ "$RUN_SONARCUBE" == "ON" ]; then
```

Comment:
Isn't it better to do `RUN_SONARCUBE=1` and in this condition `if (($RUN_SONARCUBE))`? As long as the bash version is high enough...

Reply:
Thanks for the tip!